.gitpod.yml

```diff
@@ -0,0 +1,23 @@
+# List the start up tasks. Learn more https://www.gitpod.io/docs/config-start-tasks/
+tasks:
+  - name: npm
+    init: npm install
+    command: npm run build && npm run start
+
+# List the ports to expose. Learn more https://www.gitpod.io/docs/config-ports/
+ports:
+  - port: 3000
+    onOpen: open-preview
+
+github:
+  prebuilds:
+    addBadge: true
+    addComment: false
+    addCheck: false
+    master: true
+    branches: true
+    pullRequestsFromForks: true
+
+vscode:
+  extensions:
```

Comment:
Can we also add the astro ext here?

Reply:
+1

Reply:
> Can we also add the astro ext here?

For sure. I'll add it. 🙂

Reply:
Can we also add the astro ext here?

> I didn't find any. 😅

Reply:
astro-build.astro-vscode

Reply:
Search this I think krish is talking about this one

Reply:
> Search this I think krish is talking about this one

Searched, nothing.

![image](https://user-images.githubusercontent.com/51391473/140654429-d1980dfb-49ad-4212-b616-9f63049d5317.png)

Reply:
![SCREEB](https://user-images.githubusercontent.com/72812470/140654490-8375122b-5633-4f3a-96b4-2c0025738a48.png)


But it's showing for me 🤔

Reply:
It also shows for me? 😅 Maybe we can add the id for now and see what happens :)

Reply:
> It also shows for me? 😅 Maybe we can add the id for now and see what happens :)

Gitpod won't load it if it doesn't find the extension. Anyways, let's try it out. 😄

Reply:
Oh :0, I think this is a issue at the gitpod side 🤔